hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/dc/MultiDatacenterRepositoryCommandExecutor.java

```diff
@@ -20,8 +22,17 @@ public MultiDatacenterRepositoryCommandExecutor(RepositoryManager repositoryMana
         this.rollbackEnabled = rollbackEnabled;
     }
 
+    public <T> void executeByUser(RepositoryCommand<T> command, RequestUser requestUser) {
+        if (requestUser.isAdmin()) execute(command, false, false);
```

Comment:
Is it ok if admin's failed commands will never get rolled back? Even when _hermes-management_ is not in _read-only_ mode?

Reply:
> Maybe we can also verify if hermes-management is in read-only mode here?

Yes, that would be nice. Otherwise, in a normal situation admins will not even notice that something went wrong.

Reply:
Added such verification:
```java
if (requestUser.isAdmin() && modeService.isReadOnlyEnabled()) execute(command, false, false);
```